extension/controller/backgroundScript.js

```diff
@@ -293,6 +307,25 @@ const messageListener = function(message, sender) {
             translatingAsBrowse: message.translatingAsBrowse
           });
           break;
+        case "errorCollectionConsent":
+          browser.storage.local.set({ errorCollectionConsent: message.consent });
+          Sentry.getCurrentHub().getClient()
+            .getOptions().enabled = message.consent;
+          for (const [key,] of telemetryByTab) {
```

Comment:
`key` -> `tabId` for clarity.

If telemetry is disabled, shouldn't you just wipe all telemetry?

Reply:
It seems the telemetry api already does that whenever it's switched off. @eu9ene can confirm if that's the case.

Reply:
telemetry sends data deletion ping and disables further uploading when you call `setUplpoadEnabled(false)`